### PR TITLE
feat(UIS-51): Delete employee

### PR DESCRIPTION
**Pull request by StaceyJin**
JIRA: https://champlainsaintlambert.atlassian.net/browse/UIS-51

Delete employee functionality implemented in FRONTEND, story is COMPLETED:

- updated component in user interface;
- end to end testing via Cypress implemented.

Also:
- table styling updated;
- orders page updated.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -98,7 +98,7 @@ h1 {
 }
 
 table, th, td {
-  border: 1px solid black;
+  /*border: 1px solid black;*/
```
Comment (Kam1st) [inline src/App.vue:101]: We can take this off for good I think

File: src/components/employeeDetails.vue [hunk 2]
```diff
@@ -41,11 +37,11 @@ export default {
   data() {
     return {
       currentEmployee: '',
-      message: '',
-      employeeName: '',
-      position: '',
-      dateOfHire: '',
-      status: ''
+      message: ''
+      // employeeName: '',
```
Comment (Kam1st) [inline src/components/employeeDetails.vue:41]: What are these for and do we need them?

**Review by Kam1st — APPROVED** [review]
Otherwise is fine.

File: src/App.vue (hunk 1 above)
Comment (StaceyJin) [reply src/App.vue:101]: Will be removed next chance.

File: src/components/employeeDetails.vue (hunk 2 above)
Comment (StaceyJin) [reply src/components/employeeDetails.vue:41]: They were originally added for validation, but not needed at the end, they will be removed.

**Pull request merged**